diagnostic.md

```diff
@@ -21,13 +21,21 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://10.13.105.206:3000/items
```

Comment:
Wrong URL - per the directions above, the path should be `/donuts`

Additionally, the directions tell you that you must "ask `curl` to include HTTP headers in its output" - to do this, you must have `--include` in your CURL command.